
[Executed at: Wed Mar 14 1:57:00 PDT 2018]

None
Traceback (most recent call last):
  File "/home/ccc_v1_c79523__48809/asn24778_45/asn24782_4/asnlib/submission.py", line 8, in <module>
    from q6_work import find_path
  File "./q6_work.py", line 50, in <module>
    print(get_interchange(lines))
  File "./q6_work.py", line 35, in get_interchange
    newdict[stationline.keys()] = stationline.values()
TypeError: unhashable type: 'dict_keys'
Command exited with non-zero status 1
